patroni/dcs/consul.py

```diff
@@ -38,12 +39,17 @@ class InvalidSession(ConsulException):
     """invalid session"""
 
 
-Response = namedtuple('Response', 'code,headers,body,content')
+class Response(NamedTuple):
+    code: int
+    headers: Any
```

Comment:
```suggestion
    headers: Union[Mapping[str, str], Mapping[bytes, bytes], None]
```

Based on [urllib3 source code](https://github.com/urllib3/urllib3/blob/main/src/urllib3/response.py#L533).